tests/GraphQl/Type/SchemaBuilderTest.php

```diff
@@ -147,14 +147,15 @@ public function testGetSchema(bool $paginationEnabled)
             /** @var ObjectType $objectPropertyFieldType */
             $objectPropertyFieldType = $type->getFields()['objectProperty']->getType();
             $this->assertEquals($objectPropertyFieldType->name, 'ShortName1Connection');
+            $this->assertEquals(GraphQLType::nonNull(GraphQLType::int()), $objectPropertyFieldType->getField('totalCount')->getType());
             /** @var ListOfType $edgesType */
             $edgesType = $objectPropertyFieldType->getFields()['edges']->getType();
             $edgeType = $edgesType->getWrappedType();
             $this->assertEquals($edgeType->name, 'ShortName1Edge');
-            $this->assertEquals($edgeType->getFields()['cursor']->getType(), GraphQLType::nonNull(GraphQLType::string()));
+            $this->assertEquals(GraphQLType::nonNull(GraphQLType::string()), $edgeType->getField('cursor')->getType());
```

Comment:
This has been inverted because expected value should go first.
I've also used `getField($name)` because it's shorter.